src/components/molecules/ImageGallery/styles.tsx

```diff
@@ -36,6 +36,6 @@ export const StyleHorizontalScrollWrapper = styled.div`
         background-color: transparent;
     }
     ${media.med `
-    margin-left: 20%;
+    margin-left: 18rem;
```

Comment:
`18rem` looks like a magic number 🤔

Reply:
For some reason, the gallery doesn't align perfectly center, so I had to try numbers until I found one that seemed OK. 

Should I just use % instead?